### DISCUSSION TITLE
Add support for Buzz Controllers

### DISCUSSION THREAD

**Discussion by ChosenFallen** [godotengine/godot-proposals#10827] [Engine Core]
You can currently use buzz controllers but it doesn't recognise what the controller is. 

Also, it is not possible to write any values back to the HID to light up the lights on the buzz controllers.

This would help add another variety of controller and would make it a lot easier to make quiz like games for use with these controllers.

**Comment by Dontknow09** [comment]
+1 I need this for one of my projects
